src/sass/app/components/market.scss

```diff
@@ -315,8 +315,26 @@
                     }
                     .label {
                         font-size: 20px;
-                        padding: 16px 0px;
-                        font-weight: 700;
+                        padding: 16px 0;
+                        font-weight: bold;
+                        flex: 1 0 auto;
+                        @media (min-width: 320px) and (max-width: 767px) {
+                            font-size: 20px;
+                        }
```

Comment:
Why is this needed? `font-size` is already defined at line 317

Reply:
When the screen is resize based on the rule it applies the font size of `20px` for the mobile and tablet view.